modules/apps/translation/translation-test/src/testFunctional/macros/Translations.macro

```diff
@@ -283,7 +283,7 @@ definition {
 				key_fieldValue = "${webContentDecimal}",
 				locator1 = "Translation#TARGET_LANGUAGE_FIELD");
 		}
-	}	
+	}
```

Comment:
Per our conventions please do not SF on top of your own commits

Reply:
Oh, sorry, I didn't know this before